src/Microsoft.Data.Analysis.nuget/Microsoft.Data.Analysis.nuget.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <IsPackable>true</IsPackable>
```

Comment:
I think you are going to need to set a property to tell `dotnet pack` not to include the binary from this `.csproj`. I think these 3 properties should do it:

[https://github.com/dotnet/machinelearning/blob/3d11ce42ab108d467ea6076dc14eaded57f46172/pkg/Directory.Build.props#L5-L7](https://github.com/dotnet/machinelearning/blob/3d11ce42ab108d467ea6076dc14eaded57f46172/pkg/Directory.Build.props#L5-L7)

```xml
    <NoBuild>true</NoBuild>
    <IncludeBuildOutput>false</IncludeBuildOutput>
    <IncludeSymbols>false</IncludeSymbols>
```

You will also need to move a bunch of the nuget properties from `Microsoft.Data.Analysis.csproj` to this project, since that is where nuget will read them from now.

```xml
    <VersionPrefix>0.5.0</VersionPrefix>
    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
    <Description>This package contains easy-to-use and high-performance libraries for data analysis and transformation.</Description>
    <PackageReleaseNotes>Initial preview of robust and extensible types and algorithms for manipulating structured data that supports aggregations, statistical funtions, sorting, grouping, joins, merges, handling missing values and more. </PackageReleaseNotes>
    <PackageTags>ML.NET ML Machine Learning Data Science DataFrame Preparation DataView Analytics Exploration</PackageTags>
```

Reply:
> ```
> NoBuild>true</NoBuild>
>     <IncludeBuildOutput>false</IncludeBuildOutput>
>     <IncludeSymbols>false</IncludeSymbols>
> ```

These settings break the build, you can see here: https://dev.azure.com/dnceng/public/_build/results?buildId=694421&view=results
and here: https://dev.azure.com/dnceng/public/_build/results?buildId=694435&view=results

Reply:
You'll need at least

```xml
<IncludeBuildOutput>false</IncludeBuildOutput>
```

So the .dll produced by this .csproj doesn't get included in the resulting .nupkg

Reply:
It makes sense. 
I have added and 5 of 6 jobs are successfully run. The OSX x64 Release is failing. Any idea?